DataScience/Experimentation.py

```diff
@@ -100,7 +101,7 @@ def run_experiment(command):
     
 def run_experiment_set(command_list, n_proc):
     # Run the experiments in parallel using n_proc processes
-    p = multiprocessing.Pool(n_proc)
+    p = ThreadPool(n_proc)
```

Comment:
Is there a way to set this to max(num_cores -1, 1)?

Reply:
yes, it makes sense. updated.

Reply:
Actually just had quick discussion with Susan and we decided to make it as separate change.